.circleci/config.yml

```diff
@@ -292,7 +320,13 @@ jobs:
       - run: apt-get install -y make virtualenv python-pip enchant
       - checkout
       - setup_remote_docker
-      - run: make ci-deb-tests
+      - run:
+          name: Test Debian package build
+          command: |
+            BRANCH_MATCH=$(devops/scripts/match-ci-branch.sh "^update-builder")
+            echo "match-ci-branch.sh said: ${BRANCH_MATCH}"
+            if [[ $BRANCH_MATCH =~ ^found ]]; then echo "Skipping: ${BRANCH_MATCH}"; exit 0; fi
```

Comment:
`if [[ $BRANCH_MATCH =~ ^found ]]` -> `if ! [[ $BRANCH_MATCH =~ ^found ]]` here since we only want to skip the test when we _don't_ find `update-builder` in the branch name

(btw: the `deb-tests` job isn't getting kicked off when testing from a fork since we're using circle ci's branch filtering on `/update-builder-.*/`. I think this makes sense since in practice anyone who is doing builder updates has write access to the repo.)